packages/types/src/context.ts

```diff
@@ -0,0 +1,2 @@
+export type Context = unknown;
```

Comment:
```suggestion
export type Context = Record<string, unknown>;
```

I thought the motivation to start this was realizing that a context must be a map of strings to anything else?

Reply:
No, a single context has to always be an object (see your own screenshot).

Reply:
To me this is saying the type of a single context is `unknown`, not `an object with string keys`?!